st2actions/st2actions/notifier/notifier.py

```diff
@@ -159,6 +181,22 @@ def _post_notify_subsection_triggers(self, liveaction=None, execution_id=None,
             if len(failed_routes) > 0:
                 raise Exception('Failed notifications to routes: %s' % ', '.join(failed_routes))
 
+    def _build_jinja_context(self, liveaction, execution):
+        context = {SYSTEM_KV_PREFIX: KeyValueLookup()}
+        context.update(liveaction.parameters)
```

Comment:
I brought this up on slack. This is the same story in action chain too. Prefixng here but not there would lead to inconsistency.